apps/date-component/README.md

```diff
@@ -0,0 +1,71 @@
+# Date component example
+
+This tutorial explains the steps to create a date using the [hof-component-date](https://github.com/UKHomeOfficeForms/hof-component-date)
+
+## Getting started
+
+Create a directory called date-component and use the [hof-generator](https://github.com/UKHomeOfficeForms/hof-guide/blob/master/source/documentation/getting-started.md) to create your app
+
+Currently the app loads to a name page with a field `input[type=text]`.  We will remove this and create a step that displays a date of birth.
+
+Go into the following directory `date-component-> apps -> date-component` and open the `index.js` file. Rename the first step key from `/name` to `/date-of-birth`. 
```

Comment:
I wouldn't worry too much about describing things that a user should already know how to do. i.e. generating an app and adding a field.

The starting point should be "you have an app, and have followed the basic documentation so know how to do standard things".